afs/templates/views/components/workflows/select-phys-thing-step.htm

```diff
@@ -35,7 +42,12 @@
     <!-- /ko -->
     <!-- ko if: validateThing && isPhysicalThingValid() === false -->
     <div class="tabbed-workflow-step-information-box widget-input-label" style="text-align: center; margin-top: 16px; height:fit-content; background-color: #fde9ad;">
-        The selected physical thing is invalid.  Complete the Analysis Area or Sample Taking workflow before proceeding.
+        <span data-bind="if: datasetRoute == 'non-destructive'">
+            You have to select an object with the analysis areas. Complete the Analysis Area workflow before proceeding.
```

Comment:
Perhaps it might also be good to point them back to step one - something like  "this dataset type requires an object with an analysis area; please either complete the analysis area workflow or start the workflow over and choose 
 the sample dataset type"